frontend/src/components/feedback.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+function FeedbackPage() {
+  const [feedback, setFeedback] = useState('');
+  const buttonStyle = { 
+    marginTop : "5px" ,
+    backgroundColor: "#008CBA" , 
+    color :"white" , 
+    height : "40px" , 
+    width : "150px"
+  };
+
+  const textareaStyle = {
+    width: '60%',
+    height: '200px', 
+    border: '1px solid #ccc',
+    padding: '10px' 
+  };
+
+  const feedbackstyle = {
+      marginLeft : "350px" , 
+      marginTop : "150px" , 
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setFeedback(''); 
+  };
+
+  return (
+    <div style = {feedbackstyle}>
+      <form   onSubmit={handleSubmit}>
+        <div   >
+          <h1> <b> Feedback form for Studypal </b></h1>
```

Comment:
feedback form
- subject
- category (user experience, course)
- attachments (should be optional) (user should be able to drop images: Dropzone)
- feedback message